server/api/setup.cfg

```diff
@@ -0,0 +1,13 @@
+
+[flake8]
+ignore = E127, E128, W504
+exclude = .git,__pycache__,docs/source/conf.py,old,build,dist
+max-line-length = 88
```

Comment:
Our backend CI tests run simply runs `docker-compose run api flake8` without any flags or options to `flake8` (see `/.github/workflows/Continuous_Integration_Backend.yml`) So we're gonna need to make sure that these options apply in CI, otherwise there will be differences between how testing works locally and how it works on the integration server. Maybe we could do this by copying this file in `/server/api/Dockerfile`.

Reply:
i put these ignore codes into a setup.cfg so that we don't have to change CI (i'm doing it a bit differently locally). you can either set up flake8 on your machine to use this setup.cfg or pass the ignore codes when you run it. 

there are different ways to do configs but i decided to use setup.cfg since it works for both flake8 and pytest.

btw, we should talk at some point about whether flake8 or something else is the best tool for linting moving forward but i think this approach works well for now

Reply:
Ok, I'm not attached to `flake8` if there's a better way. 

I'm gonna push a tiny change to the Dockerfile that copies the `setup.cfg` file into the api container when it's built. This just ensures that `flake8` will have access to the `setup.cfg` file regardless of the environment the command is executed in.

Reply:
i'm not sure that's necessary. flake8 seemed to pick up the setup.cfg file already but i could be missing something.

Reply:
Yeah it would work without that change because the docker-compose file adds the entire `/api` directory as a volume:

```
    volumes:
      - ./api/:/home/api
```

But if we ever want to run the api as a free-standing container outside of docker-compose (which we used to do on heroku, and might want to do later), then `flake8` wouldn't have access to the `setup.cfg` because it's never copied into the container.